### PR TITLE
Add oidc to provisioning params

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- add OIDC configuration to provisioning parameters and pass it to provisioner request

**Related issue(s)**

### CODE REVIEW

**Review by wozniakjan — APPROVED** [review]
lgtm, but I am not deeply familiar with OIDC nor KEB

File: components/kyma-environment-broker/internal/broker/plans_schema.go [hunk 1]
```diff
@@ -19,6 +19,23 @@ type ProvisioningProperties struct {
 	AutoScalerMin *Type `json:"autoScalerMin,omitempty"`
 	AutoScalerMax *Type `json:"autoScalerMax,omitempty"`
 	ZonesCount    *Type `json:"zonesCount,omitempty"`
+
+	//OIDC OIDCType `json:"oidc,omitempty"`
```
[the remark this answers is not in the document]
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/internal/broker/plans_schema.go:23]: we will need to add it later, but not now

**Review by ksputo — APPROVED** [review]
lgtm

File: docs/kyma-environment-broker/03-12-custom-oidc-configuration.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: Custom OIDC configuration
+type: Details
+---
+
+To create an SKR with custom OIDC (Open ID Connect) configuration you need to specify additional `oidc` provisioning parameters, for example:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-12-custom-oidc-configuration.md:6]:
```suggestion
To create an SKR with a custom OIDC (Open ID Connect) configuration, specify the additional `oidc` provisioning parameters. See the example:
```

**Review by klaudiagrz — DISMISSED** [review]

**Review by ksputo — APPROVED** [review]

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**